react-basic-practice/src/pages/dashboard.tsx

```diff
@@ -1,10 +1,47 @@
-import React from "react";
-import { Box, useDisclosure } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { Box } from "@chakra-ui/react";
 import { ICustomer } from "@types";
-import { Search, Button, Modal, Table, TableColumn } from "@components";
+import {
+  Search,
+  Button,
+  Modal,
+  Table,
+  TableColumn,
+  OptionMenu,
+} from "@components";
 
 const Dashboard: React.FC = () => {
-  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [isOpen, setIsOpen] = useState(false);
+  const [modalHeader, setModalHeader] = useState("Add Customer");
+  const [viewData, setViewData] = useState<ICustomer | null>(null);
```

Comment:
Pls remove null. If the customer does not exist. It will be `undefined`

Reply:
i done